FE101-Exercises/02-Top-Frequent-Elements/README.md

```diff
@@ -0,0 +1,35 @@
+# Specs
+
+Define a function `topFrequent()` that takes *an array* and *an integer k* as
+arguments and returns the *k* most frequent elements.
+
+**Example:**
+
+```javascript
+var res = topFrequent([1, 1, 1, 2, 2, 3], 2);
+console.log(res); // [1, 2]
+```
+
+***
+
+Can you make that function a function of `Array` type? It won't be tested,
+take it as a challenge.
+
+**Example:**
+
+```javascript
+var res = [1, 1, 1, 2, 2, 3].topFrequent(2);
+console.log(res); // [1, 2]
+```
+
+***
+
+**Note:**
+
+* Your code's time complexity *must not* exceed `O(n log n)`.
```

Comment:
@smddzcy will this be too advanced? maybe we can talk about big o notation in one of the intro courses